__tests__/db.test.js

```diff
@@ -30,50 +31,100 @@ describe("/api/categories", () => {
   });
 });
 
-describe('/api', () => {
-    test("GET - status: 200 - responds with endpoints.json content", () => {
-        return request(app)
-        .get("/api")
-        .expect(200)
-        .then((apiResult) => {
-            return fs.readFile(`${__dirname}/../endpoints.json`, "utf-8").then((readFileResult) => {
-                expect(apiResult.body.endpoints).toEqual(JSON.parse(readFileResult));
-            })
-        })
-    });
+describe("/api", () => {
+  test("GET - status: 200 - responds with endpoints.json content", () => {
+    return request(app)
+      .get("/api")
+      .expect(200)
+      .then((apiResult) => {
+        return fs
+          .readFile(`${__dirname}/../endpoints.json`, "utf-8")
```

Comment:
Could probably have just required it at the top of the file but this works also.